src/user_config_schema.yaml

```diff
@@ -0,0 +1,47 @@
+# top_level: bool: is this top level where dt is the same?
```

Comment:
Please could you update this? Also, please can we have this on the dev_config_schema as well?